embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/telemetry/EmbraceTelemetryServiceTest.kt

```diff
@@ -0,0 +1,51 @@
+package io.embrace.android.embracesdk.telemetry
+
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+internal class EmbraceTelemetryServiceTest {
```

Comment:
An integration test would be helpful too, as it'd verify all the separate components are working together as expected. It might be possible to adapt an existing test case to do this.

Reply:
I'll add one :)